content/components/common/AppHeader.js

```diff
@@ -72,8 +72,10 @@ function AddressBar() {
 
   const handleKeyPress = (ev) => {
     if (ev.key === "Enter") {
-      const addressMatch = address.match(/ext\+normandy:\/(.+?)$/);
```

Comment:
Oops, this is my fault. I searched for `ext+normandy`, but forgot that some cases might have it escaped like this.